blog/modules/ROOT/pages/testing-in-pkl.adoc

```diff
@@ -357,10 +371,12 @@ Running this test creates expected output:
 
 [source]
 ----
-pkl test tests/Logger.pkl
```

Comment:
Command line was also removed from here, but it doesn't start with `$` like the one above.

Reply:
Added back, added dollar